pages/dkp/kommander/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md

```diff
@@ -0,0 +1,172 @@
+
+<p class="message--warning"><strong>WARNING:</strong> If you have modified any of the <code>clusterResourceSet</code> definitions, these changes will <strong>not</strong> be preserved when running the command <code>dkp upgrade addons</code>. You must use the <code>--dry-run -o yaml</code> options to save the new configuration to a file and remake the same changes upon each upgrade.</p>
+
+Your cluster comes preconfigured with a few different core addons that provide functionality to your cluster upon creation. These include: CSI, CNI, Cluster Autoscaler, and Node Feature Discovery. New versions of DKP may come prebundled with newer versions of these addons. Perform the following steps to update these addons.
+
+<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster where you want to run the upgrade by setting the <code>KUBECONFIG</code> environment variable <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">to the appropriate kubeconfig file location</a>.</p>
+
+An alternative to initializing the KUBECONFIG environment variable is to use the `--kubeconfig=cluster_name.conf flag`.
+
+1. Replace `my-aws-cluster` with the name of the cluster.
+
+```bash
+export CLUSTER_NAME=my-aws-cluster
+dkp upgrade addons aws --cluster-name=${CLUSTER_NAME}
+```
+
+The output should be similar to:
+
+```bash
+Generating addon resources
+clusterresourceset.addons.cluster.x-k8s.io/calico-cni-    installation-my-aws-cluster upgraded
+configmap/calico-cni-installation-my-aws-cluster upgraded
+clusterresourceset.addons.cluster.x-k8s.io/tigera-operator-my-aws-cluster upgraded
+configmap/tigera-operator-my-aws-cluster upgraded
+clusterresourceset.addons.cluster.x-k8s.io/aws-ebs-csi-my-aws-cluster upgraded
+configmap/aws-ebs-csi-my-aws-cluster upgraded
+clusterresourceset.addons.cluster.x-k8s.io/cluster-autoscaler-my-aws-cluster upgraded
+configmap/cluster-autoscaler-my-aws-cluster upgraded
+clusterresourceset.addons.cluster.x-k8s.io/node-feature-discovery-my-aws-cluster upgraded
+configmap/node-feature-discovery-my-aws-cluster upgraded
+clusterresourceset.addons.cluster.x-k8s.io/nvidia-feature-discovery-my-aws-cluster upgraded
+configmap/nvidia-feature-discovery-my-aws-cluster upgraded
+```
+
+2. Monitor the pods for the core addons restarting in your cluster:
+
+```bash
+kubectl rollout status daemonset/calico-node
+--namespace calico-system
+```
+
+The output should be similar to:
+
+```bash
+daemon set "calico-node" successfully rolled out
+```
```

Comment:
This step requires the user to interact with the _managed_ cluster, whereas all other steps in this doc require the user to interact with the _management_ cluster.

Let's remove Step 2 for now. If we come up with an alternative, we'll add it here.